pkg/txn/storage/memorystorage/precommit.go

```diff
@@ -51,6 +52,7 @@ func (m *MemHandler) HandlePreCommit(meta txn.TxnMeta, req apipb.PrecommitWriteC
 		case []catalog.CreateTable:
 			for _, cmd := range cmds {
 				req := txnengine.CreateRelationReq{
+					ID:           ID(cmd.Id),
```

Comment:
What is cmd.Id?    Is this the one that HAKeeper generates?   Name is kind of comfusing.

Reply:
I mean, if I see cmd.Id, it is the seq number of the command within transaction, used for MVCC.

Reply:
renamed